vector-im/element-android
# [Poll] Unmock poll history timeline (PSG-1045, PSG-1095)

## Type of change


## Content


Adding a new SDK service `PollHistoryService`. It provides API to get only list of polls for a given room.
Using the new service to unmock data for the poll history screen.

## Motivation and context


Closes #7864 

## Screenshots / GIFs





Small testing room
<img src="https://user-images.githubusercontent.com/46314705/214540463-7181a2da-e5db-4de9-9a18-5ede546c8631.gif" width=300>

Large public room
<img src="https://user-images.githubusercontent.com/46314705/214540520-38b66064-5204-464e-86ae-c66b8663a499.gif" width=300>


## Tests



- Go to a room profile screen
- Press the "Poll history" entry
- Play with the new screen and check it loads correctly the polls of the room
- Check the screen is live, meaning new poll created after landing into the screen should appear and poll status should also be updated

## Tested devices

- OS version(s): Android 11

## Checklist
